### PR TITLE
VIH-6569 - added Phone to events and handled

### PR DESCRIPTION
**Pull request by levibutler**
VIH-6569

### Change description ###
Added `Phone` and handled in PostEvent method on VideoEventsController

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: VideoAPI/Video.API/Extensions/ConferenceEventRequestExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using VideoApi.Contract.Requests;
+using VideoApi.Domain.Enums;
+
+namespace Video.API.Extensions
+{
+    public static class ConferenceEventRequestExtensions
+    {
+        public static bool ShouldSkipEventHandler(this ConferenceEventRequest conferenceEvent) =>
```
Comment (shaed-parkar) [inline VideoAPI/Video.API/Extensions/ConferenceEventRequestExtensions.cs:8]: Really like this. Cleans up some of the empty event handlers we have.

File: VideoAPI/VideoApi.Contract/Requests/ConferenceEventRequest.cs [hunk 2]
```diff
@@ -48,5 +48,10 @@ public class ConferenceEventRequest
         /// Event reason
         /// </summary>
         public string Reason { get; set; }
+
+        /// <summary>
+        /// Can be either the phone number or "anonymous"
+        /// </summary>
+        public string Phone { get; set; }
```
Comment (shaed-parkar) [inline VideoAPI/VideoApi.Contract/Requests/ConferenceEventRequest.cs:55]: Do we handle Anonymous any differently to a phone number?

**Review by shaed-parkar — APPROVED** [review]
LGTM. Just a question regarding the behaviour if the phone number is set to Anonymous

File: VideoAPI/VideoApi.Contract/Requests/ConferenceEventRequest.cs (hunk 2 above)
Comment (levibutler) [reply VideoAPI/VideoApi.Contract/Requests/ConferenceEventRequest.cs:55]: I dont think so, nothing mentioned on the ticket

**Pull request merged**